ibus.py

```diff
@@ -105,16 +110,22 @@ def do_process_key_event(self, keyval, keycode, state):
                 self.preedit_string = self.preedit_string[:-1]
                 self.invalidate()
                 return True
-            elif keyval in num_keys:
-                index = num_keys.index(keyval)
+           elif keyval in num_keys:
```

Comment:
There's a missing space in the indentation so this doesn't run.

```
  File "/usr/share/ibus-uniemoji/ibus.py", line 113
    elif keyval in num_keys:
```

You can catch any errors by running `ibus-daemon --xim -r -v`.

Reply:
a missing space in the indentation? i'm assuming on line 113?

Reply:
Yes. Where it says `elif keyval in num_keys:` the indentation is 11 characters instead of 12.